lib/Field/FieldLabel.js

```diff
@@ -14,7 +14,8 @@ export const FieldLabel = ({ children, onChange, className }) => {
       <EditableTextWrapper
         value={children}
         className={cx(
-          `field__label block font-semi-bold leading-normal ${className}`,
+          className,
+          'field__label block font-semi-bold leading-normal -mr-2',
```

Comment:
💚Is there a way we dont have to have this minus margin 👀 ?

Reply:
No 😄 
Basically, the EditableTextWrapper has that floating pencil, which we were moving with a bit of negative margin, so the label lines up with the actions/validations around it. If you have a suggestion I'm all ears? That EditableTextWrapper was a bit of a pain in general, trying to control the size of the text area, suspicious scroll bars, and Right To Left text support... grrr.